### PR TITLE
Add docs for prometheus alert manager to corollate to grafana

### PR DESCRIPTION
**Pull request by nirsht**

### CODE REVIEW

**Comment by nirsht** [comment]
CU-2dzeh8v

File: mkdocs.yml [hunk 1]
```diff
@@ -7,44 +7,43 @@ edit_uri: https://github.com/komodorio/docs/edit/main/docs/getting-started.md
...
+      - Datadog Webhook: Integrations/datadog-webhook.md
+      - Datadog Monitor Notifications: Integrations/Datadog-Monitor-Notification.md
+      - LaunchDarkly: Integrations/LaunchDarkly.md
+      - PagerDuty: Integrations/PagerDuty.md
+      - Sentry: Integrations/Sentry.md
+      - Slack: Integrations/Slack.md
+      - Microsoft Teams: Integrations/MicrosoftTeams.md
+      - Okta: Integrations/Okta/Okta.md
+      - Azure Active Directory: Integrations/AzureActiveDirectory.md
+      - Alert Manager: Integrations/AlertManager.md
```
Comment (guymenahem) [inline mkdocs.yml:48]:
```suggestion
      - Grafana / Alert Manager: Integrations/AlertManager.md
```

File: docs/Integrations/AlertManager.md [hunk 2]
```diff
@@ -1,19 +1,26 @@
-# Prometheus Alert Manager Integration
+# Prometheus/Grafana Alert Manager Integration
 
 Integration with alert manager reports alerts also to Komodor. We connect the alert to the Kubernetes workload and display the events on the timeline.
```
Comment (guymenahem) [inline docs/Integrations/AlertManager.md:3]:
```suggestion
The Integration of Grafana / Prometheus AlertManager allows alerts triggered by AlertManager to appear on Komodor timelines. 
```

File: docs/Integrations/AlertManager.md [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
 
+---
+
 ## Installation
 
-### The Alert Manager integration involves 2 parts:
+### The Alert Manager integration involves 3 parts:
```
Comment (guymenahem) [inline docs/Integrations/AlertManager.md:9]:
```suggestion
### The Alert Manager integration involves 3 steps:
```

File: docs/Integrations/AlertManager.md [hunk 4]
```diff
@@ -1,19 +1,26 @@
...
+2. Creating webhook on the alert manager / Grafana.
+3. Adding labels to the alert.
+
+---
 
-### Enabling the integration in Komodor
+### 1. Enabling the integration in Komodor
 
-To enable the Komodor Prometheus Alert Manager integration go to [Komodor integrations page](https://app.komodor.com/main/integration) and select Prometheus Alert Manager.
+To enable the Komodor Prometheus Alert Manager integration go to [Komodor integrations page](https://app.komodor.com/main/integration) and select Prometheus/Grafana Alert Manager.
```
Comment (guymenahem) [inline docs/Integrations/AlertManager.md:19]:
```suggestion
To enable the Komodor Prometheus Alert Manager integration go to [Komodor integrations page](https://app.komodor.com/main/integration), select Prometheus/Grafana Alert Manager, and install the integration. 
```

File: docs/Integrations/AlertManager.md [hunk 5]
```diff
@@ -62,14 +69,36 @@ receivers:
...
+
+4. Click `Alerting` -> `Notification policies`.
+
+![NotificationPolicies](./img/AlertManagerGrafana/notification_policies.png)
+
+5. In the notification policies, configure komodor contact endpoint as you wish to configure alerts to Komodor.
+
+### 3. Adding labels to the alert.
+
+1. Please note to specify 2 **labels** on the alert in order to connect them to the Kubernetes workload:
```
Comment (guymenahem) [inline docs/Integrations/AlertManager.md:94]:
```suggestion

To relate the alert to the relevant workload and make the alerts visible on workloads timelines -  adding labels to alerts is required. Each alert without a label will be added to the system without mapping to a specific service.

1. Please note to specify 2 **labels** on the alert in order to connect them to the Kubernetes workload:
```

**Review by guymenahem — APPROVED** [review]

**Pull request merged**